contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -14,6 +14,19 @@ CREATE TABLE sys.babelfish_sysdatabases (
 
 GRANT SELECT on sys.babelfish_sysdatabases TO PUBLIC;
 
+-- BABELFISH_SCHEMA_PERMISSIONS
+-- This catalog is implemented specially to support GRANT/REVOKE .. ON SCHEMA ..
+-- Please avoid using this catalog anywhere else.
+CREATE TABLE sys.babelfish_schema_permissions (
+  dbid smallint NOT NULL,
+  schema_name NAME NOT NULL COLLATE sys.database_default,
+  object_name NAME NOT NULL COLLATE sys.database_default,
+  permission INT NOT NULL,
+  grantee NAME NOT NULL COLLATE sys.database_default,
+  object_type NAME NOT NULL COLLATE sys.database_default,
```

Comment:
We touched upon this thing offline, in fact we should avoid using NAME data type as it is having some peculiarity for postgres. First and foremost is that, it will always do case insensitive comparison. We understand that this catalog would not be used by customers directly but we may still consult this catalog during our operations/debugging. So lets use the data type appropriately.

Reply:
Sure. Will change all the NAME datatype to `sys.nvarchar`.